admin/dashboard/includes/class-bsf-custom-fonts-menu.php

```diff
@@ -134,6 +135,33 @@ public function update_mime_types( $defaults, $file, $filename ) {
 			$defaults['ext']  = 'otf';
 		}
 
+		if ( 'svg' === pathinfo( $filename, PATHINFO_EXTENSION ) ) {
+			// Performing SVG sanitization here.
+			$svg_content = file_get_contents( $file ); // phpcs:ignore WordPress.WP.AlternativeFunctions.file_get_contents_file_get_contents
+
+			// Use DOMDocument to parse the SVG content.
+			$dom = new DOMDocument();
+			$dom->loadXML( $svg_content );
+
+			// Allow only specific SVG elements.
+			$allowed_elements = array( 'svg', 'path', 'circle', 'rect', 'line', 'polyline', 'polygon', 'ellipse', 'g', 'text', 'use' );
```

Comment:
This might not sanitize all possible cases for the svg code. You can take hint from sanitize_svg_code function from [here](https://github.com/brainstormforce/astra/pull/5885/files#diff-fdf3a48142fb61a431c46b6ebac7a85e9604faaf015428b22451c73b529ced3f)

Reply:
Thanks @garvits-bsf for the suggestion, I will implement this.

Reply:
Updated